src/bus/driver.c

```diff
@@ -1349,6 +1349,47 @@ static int driver_method_remove_match(Peer *peer, CDVar *in_v, uint32_t serial,
         return 0;
 }
 
+int driver_reload_config_completed(Bus *bus, uint64_t sender_id, uint32_t reply_serial) {
+        Peer *sender;
+        int r;
+
+        sender = peer_registry_find_peer(&bus->peers, sender_id);
+        if (sender) {
+                _c_cleanup_(c_dvar_deinit) CDVar var = C_DVAR_INIT;
+
+                c_dvar_begin_write(&var, driver_type_out_unit, 1);
+                c_dvar_write(&var, "(");
+                driver_write_reply_header(&var, sender, reply_serial, driver_type_out_unit);
+                c_dvar_write(&var, "()");
+
+                r = driver_send_reply(sender, &var, reply_serial);
+                if (r)
+                        return error_trace(r);
+        }
+
+        return 0;
+}
+
+static int driver_method_reload_config(Peer *peer, CDVar *in_v, uint32_t serial, CDVar *out_v) {
+        int r;
+
+        if (!peer_is_privileged(peer))
+                return DRIVER_E_PEER_NOT_PRIVILEGED;
```

Comment:
Did you check this with dbus-daemon?

Reply:
Hm, I got it wrong. They allow anyone to reload config. Sounds ill-advised to me. For now, let's follow what they do, but something to discuss.